sphinx_gallery/gen_rst.py

```diff
@@ -153,12 +153,15 @@ def codestr2rst(codestr, lang='python', lineno=None):
     return code_directive + indented_block
 
 
-def extract_intro(filename, docstring):
+def extract_intro_title(filename, docstring):
```

Comment:
this naming makes me thing that it extracts *just* the title of the introduction. could we make it `extract_intro_and_title`?